### PR TITLE
Pcx 1277 page shield

### PR DESCRIPTION
**Pull request by kodster28**
Rough draft of page shield content.

Still need the following addressed:

**Me**
- Create a specific KB and link to that KB for troubleshooting issues
- Add Page Shield to the developers homepage + docs-engine config + Github build process
    - https://github.com/cloudflare/cloudflare-docs-engine/pull/318 (docs-engine config)

**Justin**
- Initial content review
- Get a Page Shield icon (currently just using Firewall icon)
- Link for the Beta signup if other folks are interested

image

image

image

image

### CODE REVIEW

File: products/page-shield/docs-config.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  product: "Page Shield",
+  pathPrefix: "/page-shield",
+  productIconKey: "firewall",
```
Comment (crwaters16) [inline products/page-shield/docs-config.js]: Guessing this is a placeholder?

File: products/page-shield/src/content/script-monitor/about/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Your application runs a lot of code, much of it that you did not write yourself. You might use internal libraries or third-party scripts to add functionality like chatbots or marketing analytics.
+
+But, more code brings greater security risks. Attackers can add or change existing scripts to steal payment or personal information. When you have a lot of code on your website — particularly code you did not write yourself — it is difficult to prevent [these attacks](https://sansec.io/what-is-magecart).
+
+## How Script Monitor works
+
+<ScriptMDefinition/>
+
+Script Monitor does this by sampling your domain traffic. Based on a percentage of your traffic, it creates a list of all scripts running on your domain.
```
Comment (crwaters16) [inline products/page-shield/src/content/script-monitor/about/index.md]: Consider leading with "based on a percentage of your traffic, script monitor...". It's a little unclear what "does this" refers to.

File: products/page-shield/src/content/script-monitor/about/index.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+Since lists are based on sampling, there may be a small delay between script deployment and appearance.
+
+Future versions of Script Monitor should improve notification and detection capabilities and add the ability to block scripts.
+
+## Who has access to Script Monitor?
+
+Script Monitor is currently in closed beta testing for Business and Enterprise clients.
```
Comment (crwaters16) [inline products/page-shield/src/content/script-monitor/about/index.md]: We should check whether to use "clients" or "customers" for consistency throughout our docs (and add the preferred term to the style guide)

**Review by crwaters16 — APPROVED** [review]
Small things, excited to see this content added to devdocs!

**Comment by kodster28** [comment]
Depends on https://github.com/cloudflare/cloudflare-docs-engine/pull/318 to build properly.

File: developers.cloudflare.com/package.json [hunk 4]
```diff
@@ -15,7 +15,7 @@
     "cloudflare-docs-engine": "git+https://github.com/cloudflare/cloudflare-docs-engine.git",
     "focus-group": "0.3.1",
     "focus-visible-polyfill": "1.0.0",
-    "gatsby": "2.24.7",
+    "gatsby": "^2.32.12",
```
Comment (pedrosousa) [inline developers.cloudflare.com/package.json]: This change might be a bit risky: the docs engine is still using version 2.24.* and I'm not sure if some of the plugins should also be updated to be fully compatible with gatsby 2.32.*.
_(Update: The other sentence I had here was incorrect.)_

**Pull request merged**